### ISSUE TITLE
`\Description` does not work. Adobe Acrobat Accessibility Checker's "Figures alternate text" check fails

### ISSUE THREAD

**Issue by Domiii** [borisveytsman/acmart#458]
Providing a `figure` with a `\Description` should add the "Figures alternate text" metadata. Sadly, that does not happen.
I even double checked that Latex does not produce a warning about missing descriptions. This is the syntax we use:

```latex
\begin{figure}[th]
  \includegraphics{x.png}
  \caption{...}
  \Description{Description here}
\end{figure}
```

However, the resulting `pdf` does not pass the "Figures altenate text" check in Adobe Acrobat Pro. It claims that all such texts are not provided. Not sure if it has to do with the fact that it is not even able to find figures before auto-tagging?

The following screenshot illustrates the problem when checking the final `pdf` containing three figures (but after auto-tagging). All of them have a `\Description`:

![image](https://user-images.githubusercontent.com/282899/153449452-3f378850-5dbe-4ef5-8a6e-06fa951fed9e.png)

**Comment by borisveytsman** [comment]
Right now this field is used only for in-house processing.  We do plan to make it work for the authors using `tagpdf` facilities

**Comment by Domiii** [comment]
While the following is not exactly part of this particular issue, it is somewhat related. (Sorry, I guess, there is no `Discussion` board on this repository, so I thought, I'd try this here.)

Just a few days ago, we made a submission and ran into [this old Adobe Acrobat bug](https://community.adobe.com/t5/acrobat-discussions/autotag-in-adobe-acrobat-dc/m-p/9858687), where its  Auto-tag feature inserted the `Times Roman` (with fallback set to some other Times New Roman) font. That font insertion caused the IEEE PDF eXpress validator to fail, since it claims that `Times Roman` is needed and missing. We were not able to fix it by bundling the font manually either.

I have 2+ questions:

* Do you have a recommendation for when stuck in this type of situation? Or is there any real discussion around accessibility in ACM? Anything more substantial than different venues linking to other venues' accessibility guidelines and telling authors to "try and make it work"?
* Also: Do you think, `autotag` will be mature enough to replace Adobe Acrobat within the next few years? (It looks like its still in early development, and there is mention of any funding.)

While this goes a bit off-topic, any response would be greatly appreciated!

**Comment by borisveytsman** [comment]
1.  ACM is working towards accessibility.  This is not easy, but we are on this path.
2.  I think for latex `tagpdf` is the way to go.  I started to experiment with it, see https://github.com/borisveytsman/acmart/tree/tagging

**Comment by Domiii** [comment]
I'm sure, you are working towards it. There is no doubt about that.
I was wondering if there is more public information out or even a public platform to discuss or ask questions?

But maybe we just have to wait until the tools catch up :)

Either way, thanks for the responses and willingness to engage!
